hyperspectral/Image_Quality_Index_Extractor

```diff
@@ -0,0 +1,119 @@
+"""
+# Name:        Blur Quality Metric (BQM)
+# Purpose:     To quantify the emergence of the blur effect which is estimated by a No-Reference Perceptual Blur Metric.
+#              References: "The Blur Effect: Perception and Estimation with a New No-Reference Perceptual Blur Metric"
+#              Function: This BQM measures sharpness or other artifacts related to the blur. 
+#              The output is raning in [0, 1]; 0 means sharp, 1 means blur.
+# Version:     1.0
+#
+# Created:     02/28/2018
+"""
+
+import numpy as np
+import scipy.ndimage
+import logging
+import json
+import spectral.io.envi as envi
+
+def rgb2gray(rgb):
```

Comment:
its not clear - why is this rgb2gray if starting with a hyperspectral image? 

And ... is there a reason not to use the rgb2gray function that already exists in scikit-image? (`skimage.color.rgb2gray(rgb)`) http://scikit-image.org/docs/stable/api/skimage.color.html#rgb2gray ... or something similar? It will be more reliable and maintainable to use existing libraries where they exist, except where there is a compelling reason to do otherwise (e.g. an existing function doesn't work or a library isn't actively maintained, etc).

Reply:
Initially, I was using rgb2gray to test the quality of an RGB image, it can be removed since we test this extractor only on the hyperspectral image.